charts/orchestrator-k8s/values.yaml

```diff
@@ -0,0 +1,156 @@
+        pluginConfig:
+          dynamicPlugins:
+            frontend:
+              janus-idp.backstage-plugin-orchestrator:
+                appIcons:
+                - importName: OrchestratorIcon
+                  module: OrchestratorPlugin
+                  name: orchestratorIcon
+                dynamicRoutes:
+                - importName: OrchestratorPage
+                  menuItem:
+                    icon: orchestratorIcon
+                    text: Orchestrator
+                  module: OrchestratorPlugin
+                  path: /orchestrator
+
+      - disabled: false
+        package: "@janus-idp/plugin-notifications@1.1.14"
+        integrity: >-
+          sha512-2hSWOrYn8r8A/tOb5hxJxucTfkuMNFVT315ucAwqlignYvRTRLsEp8vfxTfTY5TVVl71UO3SPrLlT99FnpKLyQ==
+        pluginConfig:
+          dynamicPlugins:
+            frontend:
+              janus-idp.backstage-plugin-notifications:
+                appIcons:
+                  - name: notificationsIcon
+                    module: NotificationsPlugin
+                    importName: NotificationsActiveIcon
+                dynamicRoutes:
+                  - path: /notifications
+                    importName: NotificationsPage
+                    module: NotificationsPlugin
+                    menuItem:
+                      icon: notificationsIcon
+                      text: Notifications
+                    config:
+                      pollingIntervalMs: 5000
+  upstream:
+    ingress:
+      enabled: true  # Use Kubernetes Ingress instead of OpenShift Route
+    backstage:
+      resources:
+        limits:
+          memory: 2Gi
+          cpu: 1000m
+        requests:
+          memory: 800Mi
+          cpu: 200m
+      
+      podSecurityContext:  # Vanilla Kubernetes doesn't feature OpenShift default SCCs with dynamic UIDs, adjust accordingly to the deployed image
+        runAsUser: 1001
+        runAsGroup: 1001
+        fsGroup: 1001
+
+      image:
+        tag: next
+      appConfig:
+        app:
+          baseUrl: http://localhost:9090
+        backend:
+          baseUrl: http://localhost:9090
+          cors:
+            origin: http://localhost:9090
+        integrations:
+          github:
+            - host: github.com
+              token: "INSERT VALID TOKEN HERE"
+        auth:
+          environment: development
+          providers:
+            github:
+              development:
+                clientId: "INSERT VALID CLIENT ID HERE"
+                clientSecret: "INSERT VALID CLIENT SECRET HERE"
+        catalog:
+          rules:
+            - allow:
+                [
+                  Component,
+                  System,
+                  Group,
+                  Resource,
+                  Location,
+                  Template,
+                  API,
+                  User,
+                  Domain,
+                ]
+          locations:
```

Comment:
for the notifications plugin to work, there is a need also for users.yaml

see https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/templates/rhdh-operator.yaml#L311

also, the path to the software templates of the orchestrator was changed:
```
      locations:
        - type: url
          target: https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications-backend/users.yaml
        - type: url
          target: https://github.com/parodos-dev/workflow-software-templates/blob/main/entities/workflow-resources.yaml
        - type: url
          target: https://github.com/parodos-dev/workflow-software-templates/blob/main/scaffolder-templates/basic-workflow/template.yaml
        - type: url
          target: https://github.com/parodos-dev/workflow-software-templates/blob/main/scaffolder-templates/complex-assessment-workflow/template.yaml
```

Reply:
this means I'd have to set a github token for the scaffolder to start, no? otherwise this import fails if I recall 
perhaps I'll just skip this step to get up and running fast

Reply:
I guess the PR to include a default user didn't make it?

Reply:
nope, it was rejected since the claim was in real production there will be an identity provider integrated.
therefore, there will be no place for default/random users.

Reply:
one comment about not including the users/group - the workflows will have to disable sending notifications in order not to fail. in e2e test we'll miss that part which isn't great since there were couple of bugs around notification invocation from workflows.

btw, there is a place in this chart to specify the token:
https://github.com/parodos-dev/orchestrator-helm-chart/pull/90/files#diff-b45d2f25d852e92dceb7e56b690328cb89eb1b94530d17287c8fa271ca5cd3d7R118

I modified this in the OCP chart to be nicer:
https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/values.yaml#L19

Reply:
I'm pretty sure we can mount that yamls and load them in the config from the file system instead from url
Maybe I'll give it a try in the coming days

Reply:
That will be super useful. note that github token is also required for working with software templates, however to maximize the potential of software template, we should take care of installing argocd and tekton. How much can a poor kind cluster can take? XD